src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.49.0"
+__version__ = "0.49.1"
```

Comment:
Pls increment to 0.50.0 as this involves some paramter changes of an existing adapter.

Reply:
I think the best way to determine this would be to check what's released so far and bump based on that and not using whatever's present in `main`